code/datums/components/decals/blood.dm

```diff
@@ -24,11 +24,12 @@
 	//try to find a pre-processed blood-splatter. otherwise, make a new one
 	var/index = "[REF(icon)]-[icon_state]"
 	pic = blood_splatter_appearances[index]
+
 	if(!pic)
 		var/icon/blood_splatter_icon = icon(initial(I.icon), initial(I.icon_state), , 1)		//we only want to apply blood-splatters to the initial icon_state for each object
 		blood_splatter_icon.Blend("#fff", ICON_ADD) 			//fills the icon_state with white (except where it's transparent)
 		blood_splatter_icon.Blend(icon(_icon, _icon_state), ICON_MULTIPLY) //adds blood and the remaining white areas become transparant
-		pic = mutable_appearance(blood_splatter_icon, initial(I.icon_state), I.layer)
+		pic = mutable_appearance(blood_splatter_icon, initial(I.icon_state), FLOAT_LAYER)

```

Comment:
that's the default layer for mutapp, why not just remove the arg?

Reply:
Sure thing.